articles/active-directory/develop/includes/remind-not-to-relay-token-nonaud.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Don't send your middle-tier OBO token to any non-audience party
+description: Include file warning that access tokens acquired for the middle-tier shouldn't be sent to any party except that which is identified by the audience claim.
+services: active-directory
+author: brianmel
```

Comment:
```suggestion
author: iambmelt
```

Reply:
@mmacy Should this be my employee alias? Or my GitHub account? Different usernames in my case

Reply:
@iambment Nope. The `author` metadata should be your GitHub alias. The `ms.author` value should be here MSFT alias.